src/main/io/serial_cli.c

```diff
@@ -170,7 +170,7 @@ static const char * const featureNames[] = {
 };
 
 // sync this with rxFailsafeChannelMode_e
-static char rxFailsafeModes[RX_FAILSAFE_MODE_COUNT] = { 'a', 'h', 's'};
+static char rxFailsafeModes[RX_FAILSAFE_MODE_COUNT] = { 'h', 'a', 'h', 's'};
```

Comment:
using a two dimensional array here might be clearer i think. see comment relating to `offset` below.  instead of calculating an offset you'd calculate the index into the array.  the two indexes would be a) for flight channels and b) for aux channels.